$NetBSD: patch-ae,v 1.2 2015/12/29 23:34:50 dholland Exp $

Set rpaths.
Don't clobber TCL_PACKAGE_PATH; add to it.

--- configure.in.orig	2003-11-21 14:17:14.000000000 -0600
+++ configure.in
@@ -479,7 +479,7 @@ elif test "$SHARED_BUILD" = "0" || test 
         TCL_LIB_FLAG="-ltcl`echo ${TCL_VERSION} | tr -d .`\${TCL_DBGX}"
     fi
     TCL_BUILD_LIB_SPEC="-L`pwd` ${TCL_LIB_FLAG}"
-    TCL_LIB_SPEC="-L${libdir} ${TCL_LIB_FLAG}"
+    TCL_LIB_SPEC="${TCL_LIB_SPEC_RPATH} -L${libdir} ${TCL_LIB_FLAG}"
 else
     TCL_BUILD_EXP_FILE="lib.exp"
     eval "TCL_EXP_FILE=libtcl${TCL_EXPORT_FILE_SUFFIX}"
@@ -509,13 +509,19 @@ VERSION=${TCL_VERSION}
 #--------------------------------------------------------------------
 
 if test "$FRAMEWORK_BUILD" = "1" ; then
-    TCL_PACKAGE_PATH="${libdir}/Resources/Scripts"
+    TCL_PACKAGE_PATH_ADD="${libdir}/Resources/Scripts"
 elif test "$prefix" != "$exec_prefix"; then
-    TCL_PACKAGE_PATH="${libdir} ${prefix}/lib"
+    TCL_PACKAGE_PATH_ADD="${libdir}/tcl ${prefix}/lib"
 else
-    TCL_PACKAGE_PATH="${prefix}/lib"
+    TCL_PACKAGE_PATH_ADD="${prefix}/lib/tcl"
 fi
-
+for dir in ${TCL_PACKAGE_PATH_ADD}; do
+    if test -z "${TCL_PACKAGE_PATH}"; then
+      TCL_PACKAGE_PATH="$dir"
+    else
+      TCL_PACKAGE_PATH="${TCL_PACKAGE_PATH} $dir"
+    fi
+done
 #--------------------------------------------------------------------
 #       The statements below define various symbols relating to Tcl
 #       stub support.
